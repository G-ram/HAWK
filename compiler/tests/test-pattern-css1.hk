/* passing http://www.wikicu.com/Srikar_Varadaraj */
BEGIN{}
[@#Basketball_Scouting_Report@]{
    print(this);
}
END{}
