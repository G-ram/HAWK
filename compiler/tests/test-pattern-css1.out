innerhtml=Basketball Scouting Report
nextChild=
class=mw-headline
id=Basketball_Scouting_Report
